### DISCUSSION TITLE
[Projects Beta] Unable to update issue status when creator is a ghost account

### DISCUSSION THREAD

**Discussion by mattcorr** [community/community#8287] [Projects and Issues]
I created many issues with one GitHub account. I then switched to using another GitHub account and deleted the first account.

This results in issues now being owned by a generic ghost account as shown below.

![](https://user-images.githubusercontent.com/516136/144186970-0a79e2e4-4003-4aca-9d9f-955f74c170d4.png)

Normally I would not expect this to be an issue, but now I am unable to change the status of issues created with the Ghost account. 

![](https://user-images.githubusercontent.com/516136/144187540-e6d4a72b-a2ed-4b79-b30e-60cbed305e76.gif)

If I create a new issue with my new main account, there are no problems.

Any suggestions? Can Github update the creator for existing issues to a new user account?
One workaround would be to delete all the older issues and re-create them but would prefer to only do this as a last resort.

Thanks!

**Comment by RusseII** [comment]
SAME ISSUE!
